recipes/drupal8.php

```diff
@@ -20,3 +20,11 @@
   // https://www.drush.org/latest/deploycommand/
   invoke('drush:deploy');
 })->once();
+
+task('drush:config:export', function() {
+  $releasesList = get('releases_list');
+  $prev_release = max($releasesList);
```

Comment:
This collecting `$prev_release` is not necessarily the previous or current release. Consider if you rolled back or if the next release is already happening during the deploy. There is probably a variable with info you want already in the code somewhere.